clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -21,10 +22,16 @@
  */
 final class ClusterConnectionStates {
     private final long reconnectBackoffMs;
+    private final long reconnectBackoffMax;
+    private final int reconnectBackoffExpBase;
+    private final double reconnectBackoffMaxExp;
     private final Map<String, NodeConnectionState> nodeState;
 
-    public ClusterConnectionStates(long reconnectBackoffMs) {
+    public ClusterConnectionStates(long reconnectBackoffMs, long reconnectBackoffMax) {
         this.reconnectBackoffMs = reconnectBackoffMs;
+        this.reconnectBackoffMax = reconnectBackoffMax;
+        this.reconnectBackoffExpBase = 2;
```

Comment:
Seems we don't ever allow setting this to a different value, so this should just be static constant.